events/exchange/exchange_test.go

```diff
@@ -17,23 +17,104 @@ import (
 
 func TestExchangeBasic(t *testing.T) {
 	ctx := namespaces.WithNamespace(context.Background(), t.Name())
-	testevents := []events.Event{
+	exchange := NewExchange()
+
+	t.Log("config events, All events will be published")
```

Comment:
What's the purpose of these `t.Log()` messages? They read more like comments

Reply:
done